tests/test.py

```diff
@@ -1414,20 +1429,36 @@ def test_is_label_data_valid_valid_input(self):
                    'observed-member',
                    'ratio-least-connections-member',
                    'ratio-session']
+        sat = ["automap", "{\"type\":\"automap\"}", "{\"type\":\"none\"}",
```

Comment:
This looks like a repeat of line 1383.  Does it make sense to commonize this? Also why are there two forms of automap?

Reply:
We could commonize this. There are two forms of automap because in the controller I set the default to 'automap' on the MarathonService, but I am going to refactor this so it is cleaner and just make the default {'type': 'automap'}